client/components/plans/plan-actions/index.jsx

```diff
@@ -189,14 +229,14 @@ module.exports = React.createClass( {
 			);
 		}
 
-		if ( isFreePlan( this.props.plan ) ) {
+		if ( this.shouldOfferFreeTrial() ) {
 			return (
-				<div onClick={ this.handleAddToCart.bind( null, null, 'button' ) } className={ classes } />
+				<div onClick={ this.handleClickStartTrial } className={ classes } />
```

Comment:
Is this because we have to avoid using `bind`? Another option would be to create two methods that both call this, like: `handleLinkClickStartTrial` and `handleButtonClickStartTrial`.